src/common/constants.ts

```diff
@@ -15,5 +15,5 @@ export const FILTERED_WORDS = [
 ];
 export const IMAGE_ADDRESS = 'https://raw.githubusercontent.com/cuappdev/assets/master/volume';
 export const IOS = 'IOS';
-export const IS_FILTER_ACTIVE = true;
+export const IS_FILTER_ACTIVE = false;
```

Comment:
ironic that the branch is called filter articles but filtering is off

precisely why I said to make this an environment variable in your previous commit - consider making an issue to do this in the future in a small PR